data/projects.js

```diff
@@ -421,5 +421,23 @@ coala should support generating metrics for your code.",
 		"tags" : ["Jetbrains IDEs", "Plugins"],
 		"markdown": "jetbrains-ide-plugin.md",
 		"collaborating_projects" : ["coala", "jetbrains"]
+	},
+	{
+		"name" : "Vulture dead code removal",
+		"desc" : "This project is about using vulture to automatically remove \
+				  unused python code.",
+		"requirements" : [
+				"The participant should have one patch to \
+				[vulture](https://bitbucket.org/jendrikseipp/vulture) accepted.",
+				"Being familiar with the current Bear API and what it provides.",
+				"Knowledge of ASTs and basic language grammar is a bonus."
+		],
+		"difficulty" : "medium",
+		"issues" : [],
+		"mentors" : ["jendrikseipp"],
```

Comment:
there will be one. Every project we manage will have a coala mentor. At this point we are not sure who it will be.